test/plugins/wasm_bpf/assets/bpf-sources/simple_map.bpf.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Here says the license is `Apache-2.0`. However, the license at line 48 is `Dual BSD/GPL`.

Reply:
My fault. Fixed now